### ISSUE TITLE
return value of SE_AddObjectSensor is 0 or -1

### ISSUE THREAD

**Issue by victor0777** [esmini/esmini#514]
In the function comments, return value is explained sensor ID
```
    /**
            Create an ideal object sensor and attach to specified vehicle
            @param object_id Handle to the object to which the sensor should be attached
            @param x Position x coordinate of the sensor in vehicle local coordinates
            @param y Position y coordinate of the sensor in vehicle local coordinates
            @param z Position z coordinate of the sensor in vehicle local coordinates
            @param h heading of the sensor in vehicle local coordinates
            @param fovH Horizontal field of view, in degrees
            @param rangeNear Near value of the sensor depth range
            @param rangeFar Far value of the sensor depth range
            @param maxObj Maximum number of objects theat the sensor can track
            @return Sensor ID (Global index of sensor), -1 if unsucessful

```

in the ideal_sensor.cpp, for-loop statement where j is used as loop counter, sensor id is implicitly used.
 
```

   for (int i = 0; i < 2000 && !(SE_GetQuitFlag() == 1); i++)
    {
        SE_Step();

        int objList[MAX_HITS];
        for (int j = 0; j < 2; j++)  // iterate over added sensors
        {
            int nHits = SE_FetchSensorObjectList(j, objList);
            for (int k = 0; k < nHits; k++)
            {
                printf("Sensor[%d] detected obj: %d\n", j, objList[k]);

                // Get some info of that detected object
                SE_ScenarioObjectState state;
```

in the code below, this function does not return sensor id. it is implicitly used. 
I can guess how it works, but it needs to be explicit.

```
    SE_DLL_API int SE_AddObjectSensor(int object_id, float x, float y, float z, float h, float rangeNear, float rangeFar, float fovH, int maxObj)
    {
        Object *obj = nullptr;
        if (getObjectById(object_id, obj) == -1)
        {
            return -1;
        }

        player->AddObjectSensor(object_id, x, y, z, h, rangeNear, rangeFar, fovH, maxObj);

        return 0;
    }
```

**Comment by eknabevcc** [comment]
Good find, thanks for reporting it 👍
It should now be fixed by commit d7ebe5de31cf39d4abc8c7e249f1beb54fbd9630
and will be included in next release.

**Comment by victor0777** [comment]
Thanks !
